src/api/routes/experiment.js

```diff
@@ -45,6 +45,16 @@ module.exports = {
         .catch(next);
     },
   ],
+  'experiment#patchCellSets': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => {
+      console.log('where the fuck are you');
```

Comment:
if you found it then we should remove this :P

Reply:
We could add a logger message for this request, might be useful in the future while debugging (since it is important for the pipeline and the worker)